src/components/myInfo/MyInfoWhiteBox.tsx

```
@@ -0,0 +1,25 @@
+import { View, Text } from 'react-native';
+
+import WhiteBox from '@_layouts/WhiteBox';
+
+function MyInfoWhiteBox({
+  title,
+  content,
+  styleName,
+}: {
+  title: string;
+  content: string;
+  styleName?: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)